### PR TITLE
AG-11731 - editing bundle size

### PR DESCRIPTION
**Pull request by peterjrreynolds**

### CODE REVIEW

File: community-modules/core/src/edit/editModules.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+export const EditCoreModule: Module = {
+    version: VERSION,
+    moduleName: '@ag-grid-community/edit-core',
+    beans: [EditService],
+};
+
+export const UndoRedoEditModule: Module = {
+    version: VERSION,
+    moduleName: '@ag-grid-community/undo-redo-edit',
+    beans: [UndoRedoService],
```
Comment (StephenCooper) [inline community-modules/core/src/edit/editModules.ts:23]: Does this not need to depend on the EditCoreModule?

File: community-modules/core/src/edit/cellEditors/largeTextCellEditor.ts [hunk 2]
```diff
@@ -1,27 +1,10 @@
...
-    /**
-     * Number of character columns to display.
-     * @default 60
-     */
-    cols: number;
-}
+import type { ILargeTextEditorParams } from './iLargeTextCellEditor';
 
 export class LargeTextCellEditor extends PopupComponent implements ICellEditorComp {
     private static TEMPLATE /* html */ = `<div class="ag-large-text">
```
Comment (StephenCooper) [inline community-modules/core/src/edit/cellEditors/largeTextCellEditor.ts:10]: I am checking if these static templates prevent tree shaking.

File: community-modules/core/src/edit/editModules.ts (hunk 1 above)
Comment (peterjrreynolds) [reply community-modules/core/src/edit/editModules.ts:23]: Technically no, but I've added it for consistency

File: community-modules/core/src/edit/cellEditors/largeTextCellEditor.ts (hunk 2 above)
Comment (StephenCooper) [reply community-modules/core/src/edit/cellEditors/largeTextCellEditor.ts]: This does. I have pushed a change to inline the template for this one. All other editors already do not have the static templates.

**Pull request merged**